fast-requesthandler-service/.github/workflows/fast-requesthandler-service.yml

```
@@ -0,0 +1,313 @@
+      - 'fast-requesthandler-service/**'
+      - '.github/workflows/fast-requesthandler-service.yml'
+
+env:
+  SERVICE_NAME: fast-requesthandler-service
+  DOCKER_REGISTRY: ghcr.io
+  IMAGE_NAME: gppg3/fast-requesthandler-service
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)